docs/cli/function/layers.md

```diff
@@ -0,0 +1,219 @@
+```console
+? Who should have permission to use this layer?
+> Only the current AWS account
+> Specific AWS accounts
+> Specific AWS organization
+> Public (everyone on AWS can use this layer)
+```
+
+After that, you'll be asked to configure your **layer's permission**.
+
+The **current AWS account will always have access to this layer**.
+In addition, the customer can configure access for:
+- **Specific AWS accounts**: provide a comma-separated list of AWS Account IDs to provide access to them.
+- **Specific AWS organizations**: provide a comma-separated list of AWS Organization IDs to provide access to them. *AWS Organization IDs start with `o-`.*
+- **Public**: make this layer available for everyone AWS. Anyone in AWS can reference this layer using its ARN.
+
+```console
+Next steps:
+Move your libraries in the following folder:
+[NodeJS]: amplify/backend/function/<lambda-layer-name>/nodejs/node_modules/...
+[Python]: amplify/backend/function/<lambda-layer-name>/python/dist-packages/...
+
+Include any files you want to share across runtimes in this folder:
+amplify/backend/function/<lambda-layer-name>/opt/data
+
+"amplify function update <function-name>" - configure a function with this Lambda layer
+"amplify push" builds all of your local backend resources and provisions them in the cloud
+```
+
+Your Lambda layer is ready to use after the permissions are set up.
+
+## Add shared code & assets
+
+Now that your layer is set up, you'll see a new folder with the `layer-name` added to `amplify/backend/function/`. All compatible runtimes’ folder structure are autogenerated. 
+
+
+### Add shared code
+
+<amplify-block-switcher>
+
+<amplify-block name="NodeJS">
+
+A `nodejs` folder is auto-generated for you. In there you'll find an empty `package.json` file and a `node_modules` folder. If you want to offload other node_modules you can either:
+1. `cd` into the `nodejs` folder and run `npm install <required packaged>`
+2. move all your existing function's `node_modules` content into the layer's `node_modules` folder
+
+Any node module that is in the layer's `node_modules` folder can be accessed from the function as if the node module is in the function's `node_modules` folder.
+
+*In order to take advantage of Lambda layer's for your NodeJS function, you don't even need to update your function's code!*
+
+</amplify-block>
+
+<amplify-block name="Python">
+
+A `python` folder is auto-generated for you. In there you'll find a skeleton folder structure to add your Python packages to. Add your packages to `lib/python3.8/site-packages` folder and then you can use it from within your Python function same as any other `import`. 
+
+</amplify-block>
+
+</amplify-block-switcher>
+
+### Add shared assets
+
+Any assets like large images or other files that you want to share across various functions can be placed in the `amplify/backend/function/<layer-name>/opt/` folder. Your function's code can import any assets by looking for files in the `opt/` path.
```

Comment:
should they look in /opt/ (opt within the root folder) or is it something like PATH/opt?